File: OOP/Patterns/ComplexCalc/View/IView.cs
/// <summary>
///Интерфейc IView
/// </summary>
interface IView
{
    /// <summary>
    /// Получение строки от пользователя
    /// </summary>
    /// <param name="invite">Сообщение пользователю</param>
    /// <param name="clear">Очистить терминал</param>
    /// <returns>Строка, введенная пользователем</returns>
    string GetString(string invite, bool clear = true);

    /// <summary>
    /// Вывод сообщения пользователю в консоль
    /// </summary>
    /// <param name="message">Сообщение пользователю</param>
    /// <param name="clear">Очистить терминал</param>
    void ShowString(string message, bool clear = true);

    /// <summary>
    /// Пауза выполнения до нажатие Enter
    /// </summary>
    /// <param name="invite">Сообщение пользователю</param>
    /// <param name="clear">Очистить терминал</param>
    void PressEnter(string invite, bool clear = true);
}

File: exceptions/FW/src/View/iView.java
package View;

/**
 * Интрфейс для View
 */
public interface iView
{
    
    /**
     * Получение строки от пользователя
     * @param invite Сообщение пользователю
     * @param clear Очистить терминал
     * @return Строка, введенная пользователем
     */
    String getString(String invite, Boolean clear);

   
    
    /**
     * Вывод сообщения пользователю в консоль
     * @param message Сообщение пользователю
     * @param clear Сообщение пользователю
     */
    void showString(String message, Boolean clear);

        
    /**
     * Пауза выполнения до нажатие Enter
     * @param invite Сообщение пользователю
     */
    void pressEnter(String invite);
}